docs/DeveloperGuide.md

```diff
@@ -418,20 +566,55 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
-### Deleting a person
+### Adding an entry
+
+1. Adding an entry while all entries are being shown
+
+    1. Prerequisites: None
+
+    1. Test case: `add t/e d/Lunch a/7.20 da/04-10-2022 c/Food`<br>
+       Expected: Entry is added to the specified list. Details of the added entry shown in the status message.
```

Comment:
instead of specified should be say expenditure list since its a test case?

Reply:
Resolved